Remotion/Web/Core/UI/Controls/WebTreeViewEvents.cs

```diff
@@ -25,16 +25,16 @@ namespace Remotion.Web.UI.Controls
 /// <summary> Provides data for event raised by a <see cref="WebTreeNode"/>. </summary>
 public class WebTreeNodeEventArgs: EventArgs
 {
-  private WebTreeNode _node;
+  private WebTreeNode? _node;
 
   /// <summary> Initializes an instance. </summary>
-  public WebTreeNodeEventArgs (WebTreeNode node)
+  public WebTreeNodeEventArgs (WebTreeNode? node)
```

Comment:
Please check, I don't think this should be nullable.

Reply:
There are occurrences where a nullable `WebTreeNode` is passed in. Have a look at the following:
https://github.com/re-motion/Framework/blob/0bb9bdf4fbc8b324fde86d54780a3723c47b16f7/Remotion/Web/Core/UI/Controls/WebTreeView.cs#L279
https://github.com/re-motion/Framework/blob/0bb9bdf4fbc8b324fde86d54780a3723c47b16f7/Remotion/Web/Core/UI/Controls/WebTreeView.cs#L268
(`WebTreeNodeClickEventArgs: WebTreeNodeEventArgs`)

Reply:
eeee,,,, likely a null bug in the node "selection". I think that there shouldn't be null nodes at this point (clicking / selecting a node -> the node should exist, I think)

Reply:
jira ticket please. right now, it's just a feeling that that's a bug, but if I'm wrong, we'd break things. So, we need to make this into a dedicated investigation.